src/responder/nss/nss_cmd.c

```diff
@@ -1209,38 +1209,21 @@ static errno_t sss_nss_cmd_getorigbyname_common(struct cli_ctx *cli_ctx,
     errno_t ret;
     struct sss_nss_ctx *nss_ctx;
     const char **attrs;
```

Comment:
Hi,

I think  you do not have to keep the `const` here. It was here before because of `attrs = defattrs;`, but since this is not present anymore there is no need for `const` imo. Similar with `full_attribute_list` below.

bye,
Sumit

Reply:
While this is true, I think it is better to keep the `const` for `attrs` because the pointers from `const char *cache_attrs[]` will be copied into it, and those strings are real constants (stored in non-modifiable memory).
Do you agree?

Reply:
Done.